.eslintrc.json

```diff
@@ -7,5 +7,7 @@
     "ecmaVersion": 2020,
     "sourceType": "module"
   },
+  "reportUnusedDisableDirectives": true,
+  "root": true,
```

Comment:
Could you revert this? I don't think they're really related to this PR.

I'd love to have them in a follow-up change, though!